suite-common/formatters/src/formatters/NetworkNameFormatter.ts

```diff
@@ -1,9 +1,8 @@
-import { NetworkSymbol } from '@suite-common/wallet-config';
-import { getNetworkCompatible } from '@suite-common/wallet-utils';
+import { getNetwork, NetworkSymbol } from '@suite-common/wallet-config';
 
 import { makeFormatter } from '../makeFormatter';
 
 export const NetworkNameFormatter = makeFormatter<NetworkSymbol, string>(
-    value => getNetworkCompatible(value)?.name || value,
+    value => getNetwork(value).name,
```

Comment:
only used in native [here](https://github.com/trezor/trezor-suite/blob/5b29207bb5024babd48fd5ed504c3ca3b6ee51d7/suite-native/accounts/src/components/AccountListItem.tsx#L71) & [here](https://github.com/trezor/trezor-suite/blob/89d864e8c7aff4a5b6b46b49c4cbc6d12d3ca07d/suite-native/assets/src/components/AssetItem.tsx#L116), there it is called with `NetworkSymbol` type, so LGTM